project_share/views.py

```diff
@@ -402,3 +408,74 @@ def dispatch(self, request, *args, **kwargs):
     def get_success_url(self):
         """Show confirmation."""
         return reverse('address-confirm')
+
+
+IMAGE_FILE_TYPES = ['png', 'jpg', 'jpeg', 'gif']
+
+
+class ProfileView(LoginRequiredMixin, DetailView, FormView):
+    template_name = 'project_share/user_detail.html'
+    form_class = forms.ProfileForm
+    model = User
+
+    def get_object(self, queryset=None):
+        return self.request.user
+
+    def form_valid(self, form):
+        return HttpResponseRedirect(reverse('home'))
+
+    def get_initial(self):
```

Comment:
Should include race, gender, and age if you're going to include them in the editables